cwl/workflow/convert_first.cwl

```diff
@@ -0,0 +1,44 @@
+cwlVersion: v1.0
```

Comment:
rename to something like `dict_to_interval_list.cwl` or `dict2interval_list.cwl`